tests/test_e2e_40_sdntrace.py

```diff
@@ -57,7 +57,7 @@ def get_evc(circuit_id):
 
 
     # This test may eventually fail due to https://github.com/kytos-ng/flow_stats/issues/19
-    @pytest.mark.xfail
+    # @pytest.mark.xfail
```

Comment:
@gretelliz, great to know that we can remove the xfail mark. But, let's delete the line, including the comment of the previous line that is obsolete.